pixi_skills/cli.py

```
@@ -264,5 +265,170 @@ def status(
+    if backend is None:
+        backend = _prompt_for_backend()
+
+    backend_instance = get_backend(backend)
+    try:
+        symlink_path = backend_instance.install(skill)
+        console.print(
+            f"[green]Linked '{skill_name}' for {backend.value} at {symlink_path}[/green]"
+        )
+    except ValueError as e:
+        console.print(f"[red]Failed to link '{skill_name}': {e}[/red]")
+        raise typer.Exit(1)
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)